Tests/CommandsTests/WorkspaceTests.swift

```diff
@@ -15,7 +15,7 @@ import Commands
 import PackageDescription
 import PackageLoading
 import PackageModel
-import SourceControl
+@testable import SourceControl
```

Comment:
Ok... my usual rule is `@testable` imports should only be done versus the module being tested. I would probably make it public vs use @testable, but I don't have a strong opinion.